asllib/doc/ASLTypingReference.tex

```diff
@@ -2645,18 +3852,34 @@ \section{TypingRule.SliceLength \label{sec:TypingRule.SliceLength}}
 
 \begin{emptyformal}
     \subsection{Formally}
+\newcommand\eoffset[0]{\texttt{offset}}
+\newcommand\elength[0]{\texttt{length}}
+\newcommand\toffset[0]{\texttt{t\_offset}}
+\newcommand\tlength[0]{\texttt{t\_length}}
+\begin{mathpar}
+\inferrule{
+\annotateexpr{\tenv, \eoffset} = (\toffset, \eoffset')\\
+\annotateexpr{\tenv, \elength} = (\tlength, \elength')\\
+\tstruct(\tenv, \toffset) = \TInt(\Ignore)\\
+\tstruct(\tenv, \tlength) = \TInt(\Ignore)\\
+\isstaticallyevaluable(\tenv, \elength)
+}
+{
+\annotateslices(\tenv, \texttt{Slice\_Length}(\eoffset, \elength)) = \texttt{Slice\_Length}(\eoffset, \elength')
+}
+\end{mathpar}
 \end{emptyformal}
 
 \isempty{\subsection{Comments}}
 
 \section{TypingRule.SliceRange \label{sec:TypingRule.SliceRange}}
 
   \subsection{Prose}
-      Annotating slices~\texttt{slices} in an environment~\texttt{env}
-(\texttt{annotate\_slices env slices}) results in the pair \texttt{(offset,
+      Annotating a slice~\texttt{slice} in an environment~\texttt{env}
+(\texttt{annotate\_slice env slice}) results in the pair \texttt{(offset,
 length)} and all of the following apply:
    \begin{itemize}
-   \item \texttt{slices} gives a range \texttt{(j, i)};
+   \item \texttt{slice} gives a range \texttt{(j, i)};
    \item \texttt{pre\_length} is \texttt{i +: j-i+1};
    \item \texttt{offset, length} is the result of applying TypingRule.SliceLength to \texttt{i,pre\_length}.
    \end{itemize}
```

Comment:
I fixed the syntactic sugar bitvector notations to include []

Reply:
done